src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -216,6 +216,37 @@ def test_WINNF_FT_S_SSS_10(self, config_filename):
          self.assertEqual(e.args[0], 403)
       self.assertFalse(trigger_succeed, "Full Activity Dump is expected to fail")
 
+  def generate_SSS_11_default_config(self, filename):
+    """Generates the WinnForum configuration for SSS_11. """
+    # Create the configuration for blacklisted SAS cert/key path
+
+    config = {
+      'sasCert': self.getCertFilename("blacklisted_sas.cert"),
+      'sasKey': self.getCertFilename("blacklisted_sas.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_11_default_config)
+  def test_WINNF_FT_S_SSS_11(self, config_filename):
+    """Blacklisted certificate presented by SAS Test Harness.
+       Checks that SAS UUT response with fatal alert message.
```

Comment:
The second line of a docstring should be blank (and the third not indented):
https://www.python.org/dev/peps/pep-0008/#documentation-strings

Reply:
fixed